spec/unit/config_files_inspector_spec.rb

```diff
@@ -96,231 +119,343 @@
-      expect(File.stat(cfdir).mode & 0777).to eq(0750)
+        config_file_list = subject.config_file_changes("apache2-2.4.6")
+
+        expect(config_file_list).to eq expected_apache_config_file_data
+      end
     end
 
-    it "removes config files on inspect without extraction" do
-      system = double
-      expect_inspect_configfiles(system, false)
 
-      cfdir = File.join(store.description_path(name), "config_files")
-      cfdir_file = File.join(cfdir, "config_file")
-      FileUtils.mkdir_p(cfdir)
-      FileUtils.touch(cfdir_file)
+    describe "#inspect" do
+      before(:each) do
 
-      expect(File.exists?(cfdir_file)).to be true
+        apache_config_1 = ConfigFile.new(
+            name: "/etc/apache2/default-server.conf",
+            package_name: "apache2",
+            package_version: "2.4.6",
+            status: "changed",
+            changes: ["size", "md5", "time"]
+        )
 
-      inspector = ConfigFilesInspector.new(system, description)
-      inspector.inspect(filter)
+        apache_config_2 = ConfigFile.new(
+            name: "/etc/apache2/listen.conf",
+            package_name: "apache2",
+            package_version: "2.4.6",
+            status: "changed",
+            changes: ["md5", "time"]
+        )
 
-      expect(File.exists?(cfdir_file)).to be false
-    end
+        apache_config_3 = ConfigFile.new(
+            name: "/etc/sysconfig/SuSEfirewall2.d/services/apache2",
+            package_name: "apache2",
+            package_version: "2.4.6",
+            status: "changed",
+            changes: ["size", "md5", "time"]
+        )
 
-    it "returns schema compliant data" do
-      system = double
-      expect_inspect_configfiles(system, true)
+        apache_config_4 = ConfigFile.new(
+            name: "/usr/share/info/dir",
+            package_name: "apache2",
+            package_version: "2.4.6",
+            status: "changed",
+            changes: ["deleted"]
+        )
+        apache_config_5 = ConfigFile.new(
+            name: "/usr/share/man/man1/time.1.gz",
+            package_name: "apache2",
+            package_version: "2.4.6",
+            status: "changed",
+            changes: ["replaced"]
+        )
 
-      inspector = ConfigFilesInspector.new(system, description)
-      inspector.inspect(filter, extract_changed_config_files: true )
+        iscsi_config_1 = ConfigFile.new(
+            name: "/etc/iscsi/iscsid.conf",
+            package_name: "open-iscsi",
+            package_version: "2.0.873",
+            changes: ["size", "mode", "md5", "user", "group", "time"],
+            user: "nobody",
+            group: "nobody",
+            mode: "4700"
+        )
 
-      expect {
-        JsonValidator.new(description.to_hash).validate
-      }.to_not raise_error
-    end
+        expected_apache_config_file_data = apache_config_1, apache_config_2, apache_config_3,
+					apache_config_4, apache_config_5
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Tab detected.